pkgs/development/libraries/libserdes/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, perl
+, boost
+, rdkafka
+, jansson
+, curl
+, avro-c
+, avro-cpp }:
+
+stdenv.mkDerivation rec {
+  pname = "libserdes";
+  version = "6.1.1";
+
+  src = fetchFromGitHub {
+    owner = "confluentinc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "12xm07cs0549s0y75qkj5ynyjhbj973cnx2a9zb5wj7vl9l2k66l";
+  };
+
+  outputs = [ "dev" "out" ];
+
+  nativeBuildInputs = [ perl ];
+
+  buildInputs = [ boost rdkafka jansson curl avro-c avro-cpp ];
+
+  makeFlags = [ "GEN_PKG_CONFIG=y" ];
+
+  postPatch = "patchShebangs .";
+
+  # Has configure but it’s not Autoconf so steal some bits from multiple-outputs.sh:
+  setOutputFlags = false;
+
+  preConfigure = ''
+    configureFlags="$configureFlags --libdir=''${!outputLib}/lib --includedir=''${!outputInclude}/include"
+  '';
+
+  preInstall = ''
+    installFlags="$installFlags pkgconfigdir=''${!outputDev}/lib/pkgconfig"
```

Comment:
You are using *Flags wrong. They should be arrays.

Reply:
Sorry, forgot about `*FlagsArray`. Fixed.